scrapers/wiki-flags/index.js

```diff
@@ -19,6 +19,9 @@ const scrape_wiki = async (url, out_path) => {
     let $elem = $(elem)
     let $img_a = $elem.find('a.image')
 
+    
+    if($img_a.attr('href') === undefined) return
```

Comment:
In newer node versions I think you can just do `$img_a.attr('href')?.split(...)`, right? Although, I don't know which version of node we are on.

ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
Same can apply for the Array index if it doesn't exist I think `?.[1]`

Reply:
Attempted to give this a go and hit a 404 error. Discovered that is due to the following item in the **data\wiki-flags\us.json** :
 ```
{
    "img_src": "https://www.mediawiki.org/w/index.php?title=Special:Redirect/file/Wyandot_Nation.png",
    "title": "Wyandotte Nation of Oklahoma",
    "slug": "flag-wyandotte-nation-of-oklahoma"
  },
```

Where it looks like the original url in this img_src (https://www.mediawiki.org/w/index.php?title=Special:Redirect/file/Wyandot_Nation.png) no longer has the expected image. When adding the optional chaining it seems like the url is then redefined as `https://www.mediawiki.org/w/index.php?title=Special:Redirect/file/undefined`. 

I was thinking I could add in some additional error handling to output some logging when an undefined is hit. My current commit looks like it'll remove this item once it does the file overwrite. I was wondering if you would have a preference on what to do to this item in the json if the scraper hits an undefined image.

Reply:
Ah okay, yeah, just keep it as is then